show-portal-user-info/src/main/java/com/esri/arcgismaps/sample/showportaluserinfo/screens/MainScreen.kt

```diff
@@ -0,0 +1,232 @@
+                    keyboardController?.hide()
+                    focusManager.clearFocus()
+                },
+                colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.tertiary)
+            ) {
+                Text(text = "Sign out")
+            }
+            // Load button
+            Button(
+                onClick = {
+                    onLoadPortal()
+                    keyboardController?.hide()
+                    focusManager.clearFocus()
+                },
+                colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary)
+            ) {
+                Text(text = "Load")
+            }
+        }
+    }
+}
+
+/**
+ * Displays messages to the user. This may be used to display instructions, portal info, or error messages.
+ *
+ * @param infoText the text to display
+ * @param username to display username
+ * @param email to display email
+ * @param creationDate to display creationDate
+ * @param portalName to display portalName
+ * @param userThumbnail to display userThumbnail
+ * @param isLoading whether a progress indicator should be displayed
+ * @since 200.2.0
+ */
+@Composable
+private fun InfoScreen(
+    infoText: String,
+    username: String,
+    email: String,
+    creationDate: String,
+    portalName: String,
+    userThumbnail: Bitmap?,
+    isLoading: Boolean
+) {
+    Box(
+        Modifier
+            .fillMaxSize()
+            .padding(8.dp),
+    ) {
+        LazyColumn {
+            item {
+                Text(text = infoText, modifier = Modifier.padding( bottom = 10.dp))
+                Divider()
+                Row(modifier = Modifier.padding(10.dp)) {
+                    Text(text = "Portal Name: ", fontWeight = FontWeight.Bold)
+                    Text(text = portalName)
+                }
+                Divider()
+                Row(modifier = Modifier.padding(10.dp)) {
+                    Text(text = "Username: ", fontWeight = FontWeight.Bold)
+                    Text(text = username)
+                }
+                Divider()
+                Row(modifier = Modifier.padding(10.dp)) {
+                    Text(text = "E-mail: ", fontWeight = FontWeight.Bold)
+                    Text(text = email)
+                }
+                Divider()
+                Row(modifier = Modifier.padding(10.dp)) {
+                    Text(text = "Member Since: ", fontWeight = FontWeight.Bold)
+                    Text(text = creationDate)
+                }
+                Divider()
+                Row(modifier = Modifier.padding(10.dp)) {
+                    Text(text = "Thumbnail: ", fontWeight = FontWeight.Bold)
+                    if (userThumbnail != null) {
+                        Image(
+                            bitmap = userThumbnail.asImageBitmap(),
+                            contentDescription = "User Thumbnail"
+                        )
+                    }
+                }
```

Comment:
Could you move the thumbnail to appear at the top of the `InfoScreen` composable using the rounded corner shape? 
![image](https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/assets/14892474/afbe9ee4-e095-47da-b59f-da4e12f8ab4e)